src/probnum/diffeq/odefiltsmooth/kalman_odesolution.py

```diff
@@ -40,38 +40,43 @@ class KalmanODESolution(ODESolution):
     --------
     >>> from probnum.diffeq import logistic, probsolve_ivp
     >>> from probnum import random_variables as rvs
-    >>> initrv = rvs.Constant(0.15)
-    >>> ivp = logistic(timespan=[0., 1.5], initrv=initrv, params=(4, 1))
-    >>> solution = probsolve_ivp(ivp, method="ekf0", step=0.1)
+    >>>
+    >>> def f(t, x):
+    ...     return 4*x*(1-x)
+    >>>
+    >>> y0 = np.array([0.15])
+    >>> t0, tmax = 0., 1.5
+    >>> solution = probsolve_ivp(f, t0, tmax, y0, step=0.1, atol=None, rtol=None)
```

Comment:
Also, seeing this example I wonder if I prefer the syntax now or before... I liked our own problem types^^ extracting all the content makes them a bit less useful. But I guess it's a trade-off with being more like scipy.

Reply:
Yes, I suppose it is related to the convenience function discussion we had yesterday...

Reply:
Decision: keep the "unpacked" interface; think about potential tuple unpacking for the problem classes.